### ISSUE TITLE
Stenosis wizard fails

### ISSUE THREAD

**Issue by jginternational** [KratosMultiphysics/GiDInterface#50]
After step 3 - Fluid conditions.
Stack:
```
invalid command name ""
invalid command name ""
    while executing
"$basenode selectNodes "./group\[@n='$groupid'\]""
    (procedure "AddConditionGroupOnNode" line 2)
    invoked from within
"AddConditionGroupOnNode $node $groupid"
    (procedure "spdAux::AddConditionGroupOnXPath" line 5)
    invoked from within
"spdAux::AddConditionGroupOnXPath $where "Inlet""
    (procedure "::StenosisWizard::Wizard::NextFluid" line 5)
    invoked from within
"::${wizardid}::Wizard::Next$stepId"
    (procedure "Wizard::NextEvent" line 5)
    invoked from within
"Wizard::NextEvent 2"
    (command bound to event)

```

**Comment by jginternational** [comment]
Writing mdpa also fails inlet

```
Error (l685): key "Inlet" not known in dictionary
STACK:
key "Inlet" not known in dictionary
    while executing
"dict get $dict_group_intervals $groupid"
    (procedure "writeBoundaryConditions" line 17)
    invoked from within
"writeBoundaryConditions"
    (procedure "writeConditions" line 2)
    invoked from within
"writeConditions"
    (procedure "Fluid::write::writeModelPartEvent" line 13)
    invoked from within
"Fluid::write::writeModelPartEvent"
    ("eval" body line 1)
    invoked from within
"eval $wevent"
    (procedure "write::writeAppMDPA" line 17)
    invoked from within
"write::writeAppMDPA Fluid"
    (procedure "StenosisWizard::write::writeModelPartEvent" line 3)
    invoked from within
"StenosisWizard::write::writeModelPartEvent"
    ("eval" body line 1)
    invoked from within
"eval $wevent"
    (procedure "writeAppMDPA" line 17)
    invoked from within
"writeAppMDPA $appid"
    (procedure "::write::writeEvent" line 18)
    invoked from within
"::write::writeEvent $filename"
    (procedure "::AfterWriteCalcFileGIDProject" line 4)
    invoked from within
"::AfterWriteCalcFileGIDProject {E:\JG\Escritorio\test2.gid\test2.dat} 0"


```

**Comment by jginternational** [comment]
The fail writing will be fixed when #52 is fixed.

Can be patched Remeshing manually
